### DISCUSSION TITLE
Multiples Name Fix

### DISCUSSION THREAD

**Discussion by Beinzia** [Rparadise-Team/Koriki#21] [General]
Hello community,

I discover the world of emu and I am happy to see I can emu NDS on my RG35 but someone can explain me easily how to fix that, because that pretty ugly thanks you!![image](https://github.com/Rparadise-Team/Koriki/assets/117917661/25bd1198-7af0-4c82-838c-d02dd0e618cd)

**Comment by Ninoh-FOX** [comment]
We not set support to RG35XX.

See this https://github.com/rg35xx-cfw/Koriki
